tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java

```diff
@@ -382,17 +382,17 @@ public void TestCoverageCompleteness() {
 
         final Pattern script100 = PatternCache.get("("
```

Comment:
why is this a static list? should be pulled from data (maybe lstreg?)

Reply:
Good question. I noted this in the instructions a few years ago but never investigated. See point 11 there:
- Add the script codes to TestCoverageLevel.java variable script100.
  - Why is this hardcoded here? Why does it not come from common/supplemental/coverageLevels.xml?

Thinking for a moment, I guess it can't pull it from coverageLevels.xml because that's what it wants to test, and while we can get the full list of script codes from ScriptMetadata, there is no other source for the coverage levels. But maybe we could get "all others" and test against those here? @macchiati 

I plan to merge this PR now, but we could revisit this.

Reply:
> Good question. I noted this in the instructions a few years ago but never investigated. See point 11 there:
> 
> * Add the script codes to TestCoverageLevel.java variable script100.
>   
>   * Why is this hardcoded here? Why does it not come from common/supplemental/coverageLevels.xml?
> 
> Thinking for a moment, I guess it can't pull it from coverageLevels.xml because that's what it wants to test, and while we can get the full list of script codes from ScriptMetadata, there is no other source for the coverage levels. But maybe we could get "all others" and test against those here? @macchiati

It could check  locale validity against coverage levels perhaps?

Reply:
FTR  This variable should normally not need to be updated any more after https://github.com/unicode-org/cldr/pull/1296